components/apimgt/org.wso2.carbon.apimgt.keymgt/src/main/java/org/wso2/carbon/apimgt/keymgt/service/APIKeyMgtSubscriberService.java

```diff
@@ -186,6 +186,18 @@ public OAuthApplicationInfo createOAuthApplicationByApplicationInfo(OAuthApplica
                   //set allowed grant types if grant types are not provided
                     allowedGrantTypes = oAuthAdminService.getAllowedGrantTypes();
                 }
+                if (jsonObj != null && jsonObj.has(APIConstants.JSON_CLIENT_ID)){
```

Comment:
Seems like the code is not formatted properly.

Reply:
Resolved